relay.go

```diff
@@ -281,6 +282,22 @@ func (r *Relayer) Relay(f *Frame) (shouldRelease bool, _ error) {
 	return r.handleCallReq(cr)
 }
 
+func (r *Relayer) updateCallReqContinueChecksum(f *Frame, cs Checksum) {
```

Comment:
add comments here on why we only read a single arg (e.g., if we're here, arg1 and arg2 must be complete)